Command/LoadFixtureCommand.php

```diff
@@ -117,9 +118,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $noInitialize = $input->getOption('no-initialize');
 
         if ($input->isInteractive() && !$input->getOption('append')) {
-            /** @var $dialog DialogHelper */
-            $dialog = $this->getHelperSet()->get('dialog');
-            if (!$dialog->askConfirmation($output, '<question>Careful, database will be purged. Do you want to continue Y/N ?</question>', false)) {
+            $question = '<question>Careful, database will be purged. Do you want to continue Y/N ?</question>';
+            $default = false;
+            if ($this->getHelperSet()->has('question')) {
+                /** @var $questionHelper QuestionHelper */
```

Comment:
Nowadays `/** @var QuestionHelper $questionHelper */` is more common (same in line 129)